crates/lox_core/src/bodies/cio/s06.rs

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+mod terms;
+
+use crate::bodies::cip::xy06::XY;
+use crate::bodies::fundamental::iers03::{
+    general_accum_precession_in_longitude_iers03, mean_moon_sun_elongation_iers03,
+};
+use crate::bodies::{Earth, Moon, Sun, Venus};
+use crate::math::arcsec_to_rad;
+use crate::time::intervals::TDBJulianCenturiesSinceJ2000;
+use crate::types::Radians;
+
+/// l, l', F, D, Ω, LVe, LE and pA.
+type FundamentalArgs = [Radians; 8];
+
+/// The Celestial Intermediate Origin (CIO) locator s, in radians, given the (X, Y) coordinates of
+/// the Celestial Intermediate Pole (CIP). Based on IAU 2006 precession and IAU 2000A nutation.
```

Comment:
Looks like a function doc so maybe can be rewritten a bit? 

```suggestion
/// Computes the Celestial Intermediate Origin (CIO) locator, in radians, given the (X, Y) coordinates of
/// the Celestial Intermediate Pole (CIP). Based on IAU 2006 precession and IAU 2000A nutation.
```